### PR TITLE
Updated Code of Conduct

### PR DESCRIPTION
**Pull request by subhodynamics**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
- contributors and maintainers pledge to making participation in our project
+ contributors and maintainers pledge to make participation in our project
```
Comment (mattgodbolt) [inline CODE_OF_CONDUCT.md:8]: This is a good change; thanks.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
  and our community a harassment-free experience for everyone, regardless of age,
- body size, disability, ethnicity, gender identity and expression,
+ body size, disability, ethnicity, gender identity ,and expression,
```
Comment (mattgodbolt) [inline CODE_OF_CONDUCT.md:10]: If we keep the comma, it should be on the left of the space.  Do you have a thought why the comma is helpful here? I see it as grouping "gender identity and expression" as a single concept, which seems to make sense to me, but I'm interested in your thoughts.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -40,7 +39,7 @@ Project maintainers are responsible for clarifying the standards of acceptable
 
 Project maintainers have the right and responsibility to remove, edit, or
  reject comments, commits, code, wiki edits, issues, and other contributions
- that are not aligned to this Code of Conduct, or to ban
+ that are not aligned to this Code of Conduct, or ban
```
Comment (mattgodbolt) [inline CODE_OF_CONDUCT.md:42]: The "to" here is helpful. It is the right and responsibility of maintainers... "to remove...., or to ban....", in my opinion. We can rephrase though if it's confusing.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -23,14 +23,13 @@ Examples of behavior that contributes to creating a positive environment include
 
 Examples of **unacceptable** behavior by participants include:
 
-* The use of sexualized language or imagery and unwelcome sexual attention or
- advances.
+* The use of sexualized language or imagery and unwelcome sexual attention or advances.
```
Comment (mattgodbolt) [inline CODE_OF_CONDUCT.md:26]: These seem to be just whitespace changes - are they helpful?

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -23,14 +23,13 @@ Examples of behavior that contributes to creating a positive environment include
...
 * Trolling, insulting/derogatory comments, and personal or political attacks.
 * Public or private harassment.
-* Publishing others' private information, such as a physical or electronic
- address, without explicit permission.
+* Publishing others' private information, such as a physical or electronic address, 
+  without explicit permission.
 * Other conduct which could reasonably be considered inappropriate in a
- professional setting.
+  professional setting.
```
Comment (mattgodbolt) [inline CODE_OF_CONDUCT.md:32]: This one is a fix: thanks!

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -49,9 +48,9 @@ Project maintainers have the right and responsibility to remove, edit, or
 This Code of Conduct applies both within project spaces and in public spaces
  when an individual is representing the project or its community. Examples of
  representing a project or community include using an official project e-mail
- address, posting via an official social media account, or acting as
+ address, posting via an official social media account or acting as
```
Comment (mattgodbolt) [inline CODE_OF_CONDUCT.md:51]: This is the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/). I'd like to keep it.

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -49,9 +48,9 @@ Project maintainers have the right and responsibility to remove, edit, or
...
  an appointed representative at an online or offline event.
- Representation of a project may be further defined and clarified by project
+ Representation of a project may be further defined and clarified by the project
```
Comment (mattgodbolt) [inline CODE_OF_CONDUCT.md:53]: This is a good change, thanks.

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -61,7 +60,7 @@ Instances of abusive, harassing, or otherwise unacceptable behavior may be
  [matt@godbolt.org](mailto:matt@godbolt.org). The project team will review
  and investigate all complaints, and will respond in a way that it deems
  appropriate to the circumstances. The project team is obligated to maintain
- confidentiality with regard to the reporter of an incident. Further details of
+ confidentiality concerning the reporter of an incident. Further details of
```
Comment (mattgodbolt) [inline CODE_OF_CONDUCT.md:63]: This is probably simpler, and better worded. Thanks.

**Review by mattgodbolt — COMMENTED** [review]
HI there. I'd really have ideally liked an overview of what these changes are about. We take our code of conduct seriously, and so changes to it are of course vetted. In fact, this code of conduct is 100% a copy of the https://www.contributor-covenant.org/ (v1.4). So if we make changes we'll need to attribute that.

I've made some comments on your change, but can you look at the https://www.contributor-covenant.org/ website and see if a newer version of the covenant makes more sense to you. I'd like to stick to something standard, if possible, but I do appreciate your wanting to make the code clearer.

Thanks!

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (subhodynamics) [reply CODE_OF_CONDUCT.md:26]: Yes, I just noticed few whitespaces so I removed it.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (subhodynamics) [reply CODE_OF_CONDUCT.md:10]: yes, the should have been on the left of the white space.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (subhodynamics) [reply CODE_OF_CONDUCT.md:42]: I think "or ban temporarily or permanently any contributor", sounds better, although using "to" doesn't really make any changes to the meaning, but it is just that, why use an extra preposition when we can make it mean the same with fewer words.

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (subhodynamics) [reply CODE_OF_CONDUCT.md:51]: Okay, sir.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (partouf) [reply CODE_OF_CONDUCT.md:42]: The ' or to ' is in context of 'having the right and responsibility...' - 'having the right and responsibility ban' is not a correct sentence, 'having the right and responsibility to ban' is. The 'to' should stay.

**Comment by partouf** [comment]
I will fix some thing after merging, can't do it in this PR.

**Pull request merged**